doc/support_matrix.yml

```diff
@@ -8,6 +8,12 @@ support_keys:
   - name: supported_with_caveats
     icon: ☑️
     description: supported with some caveats
+  - name: needs_testing
+    icon: 🚦
+    description: needs testing
+  - name: trivial_porting
+    icon: 🧩
+    description: needs trivial porting
```

Comment:
I'm not happy with the wording. Too burnt by my math profs stating "the rest is trivial" anx leaving me in the fog.
Is there a concrete example?